src/backend/TrafficCourts/Staff.Service/Controllers/JJController.cs

```diff
@@ -172,4 +174,176 @@ public async Task<IActionResult> SubmitAdminResolutionAsync(string ticketNumber,
         }
     }
 
+    /// <summary>
+    /// Updates each JJ Dispute based on the passed in IDs (ticket number) to assign them to a specific JJ or unassign them if JJ not specified.
+    /// </summary>
+    /// <param name="ticketNumbers">List of Unique identifiers for JJ Dispute records to be assigend/unassigned.</param>
+    /// <param name="username">IDIR username of the JJ that JJ Dispute(s) will be assigned to, if specified. Otherwise JJ Disputes will be unassigned.</param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    /// <response code="200">JJ Disputes are assigned/unassigned to/from a JJ successfully. The JJ Disputes are updated.</response>
+    /// <response code="400">The request was not well formed. Check the parameters.</response>
+    /// <response code="403">Forbidden, requires jj-dispute:update permission.</response>
+    /// <response code="404">The JJ Dispute(s) to update was not found.</response>
+    /// <response code="500">There was a server error that prevented the update from completing successfully.</response>
+    [HttpPut("Assign")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status405MethodNotAllowed)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
```

Comment:
For 500 status code, I think you need to specify it returns a `ProblemDetails`.  I looked at the swagger and it does not specify result content,

```json
          "500": {
            "description": "There was a server error that prevented the update from completing successfully."
          }
```